roles/logs_client/tasks/deploy.yml

```diff
@@ -24,6 +24,13 @@
   become: true
   notify: client_restart_rsyslog
 
+- name: Make sure that rsyslog work directory /var/spool/rsyslog is created
+  ansible.builtin.file:
+    path: /var/spool/rsyslog
+    state: directory
+    mode: '0700'
```

Comment:
```suggestion
    owner: root
    group: root
    mode: '0700'
```

Reply:
it's root by default because of `become: true`

Reply:
When the folder is missing it will get created as `root:root` due to `become: true`, but when the folder already exists and is owned by the wrong user or is in the group, that will not get fixed by `become: true`. To ensure correct state the owner and group must be added explicitly.

Reply:
1. You suggestion was already accepted yesterday.

2. > when the folder already exists and is owned by the wrong user or is in the group, that will not get fixed by `become: true`

   Indeed. In this unlikely scenario, re-running current version of current version of role would not solve the problem.
   But if we follow this scenario completely: if some random process creates `/var/spool/rsyslog` folder with wrong permissions OR changes the *ownership* of this folder > then re-running the playbook that changes the ownership back, is definitely not the answer of the problem. I could even argue that this would only cover the main problem.